data/sql/updates/pending_db_world/rev_1584283998126323200.sql

```diff
@@ -0,0 +1,37 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1584283998126323200');
+
+-- Grizzly Hills, Alliance Log Ride Start
+-- npc gordun
+DELETE FROM `gossip_menu_option` WHERE `MenuID`=21251 AND `OptionID`=0;
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (21251, 0, 0, 'Yes, I am ready to travel to Venture Bay!', 26605, 1, 1, 21252, 0, 0, 0, NULL, 0, 0);
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 27414;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 27414);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(27414, 0, 0, 1, 62, 0, 100, 0, 21251, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Gordun - On Gossip Option 1 Selected - Close Gossip'),
+(27414, 0, 1, 2, 61, 0, 100, 0, 0, 0, 0, 0, 0, 85, 48622, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Gordun - On Gossip Option 1 Selected - Invoker Cast \'Alliance Log Ride 01 Begin\''),
+(27414, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 85, 48621, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Gordun - On Gossip Option 1 Selected - Invoker Cast \'Log Ride Alliance 01\'');
+
+-- Grizzly Hills, Horde Log Ride Start
+-- npc darrok
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 27425;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 27425);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(27425, 0, 0, 1, 62, 0, 100, 0, 9528, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Darrok - On Gossip Option 0 Selected - Close Gossip'),
+(27425, 0, 1, 2, 61, 0, 100, 0, 0, 0, 0, 0, 0, 85, 48960, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Darrok - On Gossip Option 0 Selected - Invoker Cast \'Horde Log Ride 01 Begin\''),
+(27425, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 85, 48961, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Darrok - On Gossip Option 0 Selected - Invoker Cast \'Log Ride Horde 00\'');
+
+-- quest Metzen the Reindeer
```

Comment:
@grimgravy why is this related with the PR?

Reply:
It actually makes no sense either because it is exactly like the DB is now, except with a different MenuID.
```sql
SELECT * FROM gossip_menu_option WHERE menuid IN (21251, 6763)
```

Reply:
Personally I think the Metzen changes should be in its own PR if these changes are necessary since it has no connection to the original intent of this pr? 🤷‍♂

Reply:
@BarbzYHOOL, had two types of creature using the same menuid (21251). Conflicts between them.

Reply:
The change is ok, but it does not have anything to do with the main issue of the PR, which is why I wondered if it could be in a separate PR ^^

Reply:
I agree with @Kitzunu also it is not good to push things to a PR that has already been tested, unless really important.

@grimgravy  can you please revert this PR to its original state? and push the other changes in another PR

Reply:
@grimgravy can you move all after ` -- the change was necessary because of the conflicts between MenuID existing in the DB.` to another PR?

Reply:
@FrancescoBorzi, has been settled

Reply:
nothing has been settled

Reply:
@FrancescoBorzi and @BarbzYHOOL if I do that, I'm gonna have to change here too.

Reply:
explain why clearly please

what was the conflict BEFORE this PR

Reply:
@grimgravy I can revert it for you

Reply:
![alli](https://user-images.githubusercontent.com/2193800/76796725-cecf1300-67aa-11ea-9a86-55b750631161.png)

Reply:
if someone understands then do something plz (shin?)